client/src/utils/urlify.test.js

```diff
@@ -9,6 +9,36 @@ describe('urlify', () => {
         'lisboa-e-otima-a-beira-mar',
       );
     });
+
+    it('should keep leading spaces & convert to hyphens if supplied', () => {
+      expect(urlify('  I like _ßpaces')).toBe('-i-like-_sspaces');
+    });
+
+    it('should allow a supplied locale to re-order downcode mapping', () => {
+      // mix of characters that are transliterated differently across Russian/Ukrainian
+      const chars = 'Ц-йи-Ґ-Х-щ';
+      const ruTransliteration = 'c-ji-g-x-shh';
+      const ukTransliteration = 'ts-iy-g-kh-shch';
+      expect(urlify(chars)).toEqual(ruTransliteration); // by default the Russian values come through with priority
+      expect(urlify('Ц-йи-Ґ-Х-щ', { locale: 'ru' })).toEqual(ruTransliteration);
+      expect(urlify('Ц-йи-Ґ-Х-щ', { locale: 'uk' })).toEqual(ukTransliteration);
+
+      // another example
+      expect(urlify('Георгій', { locale: 'uk-UK' })).toEqual('heorhii');
+      expect(urlify('Георгій', { locale: 'en' })).toEqual('georgij');
+      expect(urlify('Георгій')).toEqual('georgij');
+      expect(urlify('Георгій', { locale: 'ru' })).toEqual('georgij');
+    });
+
+    it('should support locale with normal urlify whitespace handling', () => {
+      const title = 'жйи Проєкт   Герой Їжак Георгій ';
```

Comment:
I don't know if it's important, but there's no meaningfulness here. It's just a bunch of letters

Reply:
Correct, it's just an example to test the differences. 

In my mind 'real world' examples are nice to have but the primary goal is to test the differences and validation of expected behaviour.